packages/components/src/logos/jetpack-logo/index.tsx

```diff
@@ -64,22 +85,40 @@ const LogoPathSize32Monochrome = () => (
 	</>
 );
 
+/**
+ * JetpackLogo component
+ *
+ * The 'colorVariant' prop controls the color scheme of the logo: 'color', 'black', or 'white'.
+ * - 'color': Uses Jetpack green and white (default).
+ * - 'black': Renders the logo in black and white.
+ * - 'white': Renders the logo in white and black (for dark backgrounds).
+ *
+ * The 'monochrome' prop, when true, overrides the 'colorVariant' and renders the logo and text in a single color scheme:
+ * - The icon is white, and the text and triangles are black, regardless of the 'colorVariant'.
+ */
```

Comment:
Having props that conditionally override other props can cause confusion in the consumers of the component. A better alternative is to have a single "variant" prop with all the options.

Also, I'm not convinced about `white` and `black` variants, as they don't take theming into account. For example, would a white variant change to black on a dark theme?

Also, we actually don't use pure black for our text color, meaning that the variant name may be misleading, and that currently the text and the logo don't use the same color in the `black` variant.

Maybe we could just have a `currentColor` variant and use whatever text color is inherited from the CSS cascade (ie. `currentColor`)?

(cc @jameskoster )